kld/src/cli/client.rs

```diff
@@ -29,9 +29,7 @@ impl Api {
     pub fn new(host: &str, cert_path: &str, macaroon_path: &str) -> Result<Api> {
         let macaroon = read_file(macaroon_path)?;
         let cert = Certificate::from_pem(&read_file(cert_path)?)?;
-        // Rustls does not support IP addresses (hostnames only) so we need to use native tls (openssl). Also turn off SNI as this requires host names as well.
```

Comment:
Is the sentence about rustls no longer relevant?

Reply:
It uses ip addresses now.